README.md

```diff
@@ -87,6 +88,12 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 - "List paths in my Data Lake file system"
 - "Show my key-value pairs in App Config"
 
+### 📈 Azure Workbooks
+- "List my Azure workbooks in resource group 'mygroup'"
+- "Show me details about workbook 'myworkbook'"
```

Comment:
the build fails because it doesn't like the spelling of `mygroup` and `myworkbook` ?  could always just say "in resource group 'workbooks'" and  "show me details about workbook 'incident troubleshooting'" or something?